keras/layers/layer.py

```diff
@@ -1146,6 +1146,12 @@ def load_own_variables(self, store):
         Args:
             store: Dict from which the state of the model will be loaded.
         """
+        self._check_load_own_variables(store)
```

Comment:
You can leave out `layer.py` changes and only override the Dense/EinsumDense `load_own_variables` method.

Reply:
Fixed